sql/2022/javascript/fid.sql

```diff
@@ -15,8 +11,7 @@ base AS (
     `chrome-ux-report.materialized.device_summary`
   WHERE
     device IN ('desktop', 'phone') AND
-    date IN ('2022-06-01') AND
-    IS_NON_ZERO(p75_fid, p75_fid_origin)
```

Comment:
We should still filter out `NULL` FID values. Can do that in the `APPROX_QUANTILES` functions:

```sql
APPROX_QUANTILES(p75_fid, 1000 IGNORE NULLS)[OFFSET(percentile * 10)] AS p75_fid
```